cli-addService

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env node
+
+// Adds a new service; map the public path with the service URL and the application Id.
+
+var util = require('./util'),
+    program = require('commander'),
+    config = require('./config'),
+    logger = require('./log');
+
+var db = require(config.database.type);
+var urlRegExp = /(ftp|http|https):\/\/(\w+:{0,1}\w*@)?(\S+)(:[0-9]+)(\/|\/([\w#!:.?+=&%@!\-\/]))?/;
+
+program
+    .option('-c, --context-broker', 'the service is an Orion Context broker service')
+    .description(
+        'Adds a new service for monitoring. Binds the public path with the URL specified and the application ID.\n' + 
+        '  The public path valid patterns are the followong: \n' +
+        '    * /publicPath: only the first part of the path \n' +
+        '    * /this/is/the/final/resource/path?color=Blue&shape=rectangular`: the complete resource path (absolute path).\n' +
+        '       In this case, the proxy will use this path to make the request. Use this type of public paths to register \n' + 
+        '       URLs with query strings. For instance, a public path such as `/public/path` is not valid.')
+    .parse(process.argv);
+
+if (program.args.length !== 3) {
+    logger.error('Invalid number of arguments. Usage: ./cli addService [-c] <publicPath> <URL> <appId>');
+    console.log(program.help());
+
+} else {
+
+    var contextBroker = program.contextBroker !== undefined ? true : false;
+    var publicPath = contextBroker ? program.args[2] : program.args[0];
+    var url = contextBroker ? program.args[0] : program.args[1];
+    var appId = contextBroker ? program.args[1] : program.args[2];
```

Comment:
I don't understand why the external path and the URL are taken from different args if the service is a context broker